Nelwell/lmn
# Add user info

Added additional information about each user to their profile page. 

To test:
1. Make migrations/migrate
2. In the admin console, create a new badge
3. In the admin console, create a new Profile, which will have a one-to-one relation to a user. Make sure the user it's based on 
   has a first name, last name, and email
4. In the application, click on the link to any user's username to view their information

@John151 @Nelwell @abdiawali @PickleEric
